src/app/OrderBook.tsx

```diff
@@ -6,25 +6,59 @@ export function OrderBook({ orders }: { orders: OrderbookLine[] }) {
   //returns simple orderbook of buys/sells
   const adexState = useContext(AdexStateContext);
   const { buys, sells } = adexState.currentPairOrderbook;
+  // return (
+  //   <>
+  //     {orders.map(
+  //       (
+  //         item: {
```

Comment:
But also please note, that the work for OrderBook is in another branch: `order-book`, let's keep this branch about new Orders creation